VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.html

```diff
@@ -128,66 +128,40 @@ <h2 class="govuk-heading-s">Participants ({{ nonJudgeParticipants.length }})</h2
       </div>
     </div>
 
-    <div *ngIf="litigantInPerson" class="vh-border">
-      <div *ngFor="let participant of individualParticipants; let lastItem = last" class="govuk-grid-row govuk-!-margin-bottom-1">
+    <div class="vh-border">
+      <div *ngFor="let participant of nonJudgeParticipants; let lastItem = last" class="govuk-grid-row govuk-!-margin-bottom-1">
         <div id="p-row-{{ participant.id }}-participant" class="govuk-grid-column-full govuk-!-margin-top-2">
-          <!-- if hearing has litigant in person -->
-          <div *ngIf="litigantInPerson">
-            <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
-              {{ participant.name }}
-            </p>
-            <p id="p-{{ participant.id }}-hearing-role-participant" class="govuk-body-m govuk-!-margin-bottom-3">
-              {{ participant.hearing_role }}
-            </p>
-            <p
-              *ngIf="!isCaseTypeNone(participant)"
-              id="p-{{ participant.id }}-case-type-group-participant"
-              class="govuk-body-m govuk-!-margin-bottom-3"
-            >
-              {{ participant.case_type_group }}
-            </p>
-          </div>
 
-          <div class="govuk-body-s govuk-!-margin-bottom-3">
-            <label
-              id="p-{{ participant.id }}-status-participant"
-              class="govuk-body-s label-status label-status--{{ getParticipantStatusCss(participant) }}"
-            >
-              {{ getParticipantStatus(participant) }}
-            </label>
-          </div>
+          <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
+            <img *ngIf="participant.hasInterpreterLink" class="vh-interpreter-icon" src="/assets/images/interpreter.png" alt="Interpreter" />
```

Comment:
is this the same icon as from the fonts @shaed-parkar  ?

Reply:
This is the Icon put on the story by Emma

Reply:
We just had a chat in the interpreter channel and it was advised we switch over to the free icons

Reply:
where are the icons, so I can change them and get this gd PR complete

Reply:
I found your comment about fontawesome so will use that

Reply:
now uses font awesome language icon with x2 size